tests/authorization.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {describe} from "mocha";
+import { Browser, Builder, By, Key, until, WebDriver } from "selenium-webdriver"
+import {expect} from "chai";
+
+describe('[SHARELANE-2] Authorization', async () => {
+    it('should authorize user', async () => {
```

Comment:
Pleease use beforeEach / afterEach() mocha methods instead of try / catch for driver initialisation / disposing

Reply:
Fixed